pkg/jsonnet/eval.go

```diff
@@ -60,7 +58,7 @@ func MakeVM(opts Opts) *jsonnet.VM {
 // result in JSON form. It disregards opts.ImportPaths in favor of automatically
 // resolving these according to the specified file.
 func EvaluateFile(jsonnetFile string, opts Opts) (string, error) {
-	jpath, _, _, err := jpath.Resolve(filepath.Dir(jsonnetFile))
+	jpath, _, _, err := jpath.Resolve(jsonnetFile)
```

Comment:
You call `jpath.Resolve` here with `jsonnetFile`, but from what I can see it requires a `workdir` (directory), right?

Reply:
Yeah, variable naming is a bit awkward all over the place, the code mostly assumes directories as the start point but it could be both now. I would like to address consistent naming in another PR if that is fine with you?

Reply:
Example:
```
 func eval(dir string, opts jsonnet.Opts) (raw interface{}, env *v1alpha1.Config, err error)
````

The first argument is also a dir, but might be a file in this case. I'd like to address this after #403 with a less intrusive refactoring (only naming, nothing changing anything functional).